crates/fuel-core/src/graphql_api.rs

```diff
@@ -80,8 +80,8 @@ impl Default for Costs {
 }
 
 pub const DEFAULT_QUERY_COSTS: Costs = Costs {
-    // TODO: The cost of the `balance` and `balances` query should depend on the
-    //  `OffChainDatabase::balances_enabled` value. If additional indexation is enabled,
+    // TODO: The cost of the `balance`, `balances` and `coins_to_spend` query should depend on the
```

Comment:
While the TODO comment is not introduced in this PR, I'd like to know if we have any issue for this that we can link?

Reply:
Yes, [issue](https://github.com/FuelLabs/fuel-core/issues/2496) has been created, but not linked. Updated in 65f701d7c05e0ddc26c8dc7adc97211bdfeff152

Reply:
Nice thanks!